objects/official/scenery_wall/official.scenery_wall.post_flipped.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "official.scenery_wall.post_flipped",
+    "authors": [
+        "OpenRCT2 developers",
+        "Simon Foster"
+    ],
+    "version": "1.0",
+    "sourceGame": "official",
+    "objectType": "scenery_wall",
+    "properties": {
+        "isBanner": true,
+        "hasPrimaryColour": true,
+        "height": 4,
+        "price": 70
+    },
+    "images": [
+        "$RCT2:OBJDATA/WALLPOST.DAT[6..7]",
+        "",
```

Comment:
Can you explain these empty images?

Reply:
Scenery walls seem to reserve image slots for slope sprites even if they're not enabled by the wall object. You can see this in the vanilla post's images when you export them.

I recall you mentioned that scenery walls will still display slope images even with the flag not being set, so i assume this is how that works.

Exported vanilla post images, there's no empty images after 7 so the empty images only exist for the first set of sprites (0 thru 5).
![Screenshot_select-area_20240220050849](https://github.com/OpenRCT2/objects/assets/42477864/a1e296db-cb67-4b7d-bfcf-0f1027d19481)